ecosystem/typescript/sdk/src/utils/property_map_serializer.test.ts

```diff
@@ -0,0 +1,23 @@
+import { getPropertyValueRaw } from "./property_map_serializer";
+import { bcsSerializeBool, bcsSerializeStr, bcsSerializeU128, bcsSerializeU8, bcsSerializeUint64, Bytes } from "../bcs";
+import assert from "assert";
+
+test("test property_map_serializer", () => {
+  var is_same: (array1: Bytes, array2: Bytes) => boolean = function (array1, array2): boolean {
+    return (
+      array1.length == array2.length &&
+      array1.every(function (element, index) {
```

Comment:
by convention, we should use an arrow function here. `(element, index) => {`

Reply:
updated